pyproject.toml

```diff
@@ -6,8 +6,8 @@ requires = [
     "numpy==1.16.5; python_version=='3.6' and platform_system!='AIX'",
     "numpy==1.16.5; python_version=='3.7' and platform_system!='AIX'",
     "numpy==1.17.3; python_version=='3.8' and platform_system!='AIX'",
-    "numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'",
-    "numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'",
+    "numpy==1.16.5; python_version=='3.6' and platform_system=='AIX'",
+    "numpy==1.16.5; python_version=='3.7' and platform_system=='AIX'",
     "numpy==1.17.3; python_version=='3.8' and platform_system=='AIX'",
```

Comment:
Since they all match now, is there any reason not to drop the `platform_system` clause entirely?

Reply:
thats fair, but keeping them different helps remember hte history a little bit.

Reply:
I sort of see your point, but having it in two places is why these didn't get updated in the first place. Better to have a "single source of truth".

Reply:
you are right. the history isn't even that well conveyed in the code. So there isnt' really any advantage.